cli/tests/test_config_command.rs

```diff
@@ -592,11 +592,10 @@ fn test_config_set_for_user_directory() {
     let output = test_env.run_jj_in(".", ["config", "set", "--user", "test-key", "test-val"]);
     insta::assert_snapshot!(output, @r"
     ------- stderr -------
-    Error: Cannot determine config file to edit:
-      $TEST_ENV/config/config0001.toml
-      $TEST_ENV/config/config0002.toml
+    1: $TEST_ENV/config/config0001.toml
+    2: $TEST_ENV/config/config0002.toml
+    Choose a config file (default 1): 1
     [EOF]
-    [exit status: 1]
     ");
```

Comment:
nit: Can you insert snapshots of these config files to ensure that the first file is updated?

Reply:
Done!